packages/firestore/src/local/local_documents_view.ts

```diff
@@ -51,7 +51,8 @@ import { SortedMap } from '../util/sorted_map';
 
 import { DocumentOverlayCache } from './document_overlay_cache';
 import { IndexManager } from './index_manager';
-import { MutationQueue } from './mutation_queue';
+import { getAllMutationBatchesAffectingDocumentKeys } from './indexeddb_mutation_queue';
```

Comment:
As stated above, we should not import from IndexedDB or Memory persistence here. We should only use their shared APIs.